blockchain/validators/src/metadata_key/config.rs

```diff
@@ -0,0 +1,268 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Config for validators.
```

Comment:
My intuition is for objects that validate - I feel like we should organize it in a submodule like:

blockchain/validators/src/types

And then we'd have
blockchain/validators/src/types/metadata_key.rs
blockchain/validators/src/types/avr_records.rs

Thoughts?

Reply:
It's unclear to me where the combined metadata validator would go in that scheme.

I just restructured to: having a `mc_blockchain_validators::MetadataValidator` that composes `mc_blockchain_types::BlockMetadata::verify()`, `mc_blockchain_validators::metadata::KeyRangeValidator` and soon an `mc_blockchain_validators::metadata::AvrHistoryValidator`. How does that sound?

Reply:
I can see us adding other blockchain validators in the near future. If not, we can inline the `mod metadata` later.